core/controllers/acl_decorators.py

```diff
@@ -3171,11 +3496,11 @@ def test_can_access(
                     verified_classroom_url_fragment,
                     url_substring),
                 self.GET_HANDLER_ERROR_RETURN_TYPE)
-            return
+            return None
 
         topic_id = topic.id
         topic_rights = topic_fetchers.get_topic_rights(
-            topic_id, strict=False)
+            topic_id, strict=True)
```

Comment:
> What is the effect of this change? 

Previously, If `topic_fetchers.get_topic_rights` somehow results into the None value, then that value is assigned to `topic_rights` and in further code, we are doing `topic_rights.topic_is_published` so this thing will become an error ( a python error ). So just to prevent this thing and to throw explicit error I used true here ( and to strict the type also ). 

> Does the docstring need to be updated with a new exception?

I don't think so, because `strict=True`'s exception belongs to `get_topic_rights` and those exceptions are already documented there. If we write `get_topic_rights`'s exceptions here then I think we are just repeating ourselves.

Reply:
Thanks for explaining. The change sounds fine, but I actually think you should update the docstring. If you think about it from the point of view of a caller of this function (who doesn't look at its internals), this is part of the function's contract, so it's worth documenting.

Reply:
docstring updated!